umbraco-cloud/getting-started/migrate-existing-site/README.md

```diff
@@ -196,15 +196,16 @@ In this next part, it is time to generate the so-called UDA-files for all your p
 
 For more details about UDA files, read the [UDA Files](../../set-up/power-tools/generating-uda-files.md#what-are-uda-files) article.
 
+Your local Cloud project will need [Umbraco Deploy](https://www.nuget.org/packages/Umbraco.Deploy.OnPrem) installed, you can check your .csproj file to see if this is already installed. If not installed, install this and check your site is working as expected.
+ 
 * Make sure the folder `/Umbraco/Deploy/Revision` on your Umbraco Cloud project is empty
   * If you have any files in the folder, you can safely remove those at this point
 * Start your local cloud project
 * Go to the settings section in your project
-* Select `Extract Schema To Data Files` Command from the **Deploy Operations** drop-down and click **Trigger Operation**.
+* Go to the Deploy tab
+* Under **Deploy Operations** > **Export Schema To Data Files** click **Export Schema**.
   * Generating the UDA files may take a while, depending on the extent of your project
-  * You will see a `Last deployment operation completed` status once the export is done
-  * Run `Schema Deployment From Data Files` to check that the UDA files have been generated correctly
-  * When you see a `Last deployment operation completed` status, it means everything is working as expected
+  * When you see a `Last deployment operation completed` status at the top of this page, it means everything is working as expected
 * You should now see that your `/Umbraco/Deploy/revision` folder has been populated with UDA files corresponding to your project's metadata
 
 ![Run echo > deploy-export](images/deployDashBoard.png)
```

Comment:
If you add the above change, Could you maybe highlight both buttons, so the one for `update Umbraco Schema` and` export schema `? 😊

Reply:
@jonat123 I have made that change, could you check I've got the right of the stick here :)

Reply:
Cool!

I think I was a bit unclear here 🙈 What I was thinking was that in the image, it should highlight both buttons and not only the export schema button 😊

Reply:
Ah, got you, okay that will take me a bit of time as ill have to get the code up and running again, one for next week @jonat123